.editorconfig

```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see http://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 4
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.md]
+max_line_length = off
```

Comment:
why off? why not 80?

Reply:
So people can choose what line length they want and how wrapping should be handled to their liking. Java code conventions don't specify the line lenght (unlike say PEP), so I see no reason to enforce this. We are not  coding on paper after all.

Reply:
Ok, I thinked, that Java code conventions specify the line lenght...